qt/applications/workbench/workbench/app/mainwindow.py

```diff
@@ -98,7 +98,8 @@ def qapplication():
         UsageService.setApplicationName(APPNAME)
 
         # removes the ? button from the title bar of dialog windows
-        app.setAttribute(Qt.AA_DisableWindowContextHelpButton)
+        if qVersion() >= '5.10':
```

Comment:
Comparing '5.9.5' >= '5.10' returns true as it checks the single characters and does not treat it as a number, so the Workbench still crashes